source/beamline/AMControl.h

```diff
@@ -633,6 +663,26 @@ protected slots:
 	/*! If isEnum() returns true and this is not specified by a subclass implementation, the regular enumNames() will be assumed to apply for both move() and value(). moveEnumNames() will return enumNames(). */
 	void setMoveEnumStates(const QStringList& enumStateNames) { if(moveEnumNames_ == enumStateNames) return; moveEnumNames_ = enumStateNames; emit enumChanged(); }
 
+	/// Sets the move start value.
+	void setMoveStart(double newValue);
+	/// Updates the move start value.
+	virtual void updateMoveStart();
```

Comment:
All three values should be updated before the move starts, so that we use _this_ move's information. The move value is what is updated to report the move progress (ie it could change every time the control value changes). Does that answer your question?